synapse/lib/stormtypes.py

```diff
@@ -6714,7 +6714,8 @@ class LibView(Lib):
          'type': {'type': 'function', '_funcname': '_methViewAdd',
                   'args': (
                       {'name': 'layers', 'type': 'list', 'desc': 'A list of layer idens which make up the view.', },
-                      {'name': 'name', 'type': 'str', 'desc': 'The name of the view.', 'default': None, }
+                      {'name': 'name', 'type': 'str', 'desc': 'The name of the view.', 'default': None, },
+                      {'name': 'worldreadable', 'type': 'boolean', 'desc': 'Grant read access to the `all` role.', 'default': False, },
```

Comment:
Do we need to add a `--worldreadable` to the `view.add` command? Or leave it as-is and let people manually opt into this when constructing a view via API?

Reply:
Not sure I understand your comment. I did add a `--worldreadable` to `view.add`. Did I miss something somewhere else?

Reply:
You're good! i misread :)